modules/solid_mechanics/doc/content/source/materials/CompositePowerLawCreepStressUpdate.md

```diff
@@ -0,0 +1,41 @@
+# Power Law Creep Stress Update
+
+!syntax description /Materials/CompositePowerLawCreepStressUpdate
+
+## Description
+
+!include modules/solid_mechanics/common/supplementalRadialReturnStressUpdate.md
+
+Similar to [PowerLawCreepStressUpdate](/PowerLawCreepStressUpdate.md), with the addition of multiple phases, increment of inelastic strain is computed from the creep rate in this class.
+
+\begin{equation}
+  \label{eq:power_law_creep}
+  \dot{\epsilon} = \sum_{i=1}^N h_i \left(A_i \left( \sigma^{trial}_{effective} - 3 G \Delta p \right)^n_i exp \left( \frac{-Q_i}{RT} \right)\right) \left(t - t_o \right)^m
+\end{equation}
+
+where subscript i denotes phase specific material property, $h$ is weighting function to interpolate different phases, $A_i$ is the power law creep coefficient, also known as Dorn's Constant, $\sigma^{trial}_{effective}$ is the scalar von Mises trial stress, $G$ is
+the isotropic shear modulus, $Q$ is the activation energy, $R$ is the universal
+gas constant, $T$ is the temperature, $t$ and $t_o$ are the current and initial
+times, respectively, and $n$ and $m$ are exponent values.
+
```

Comment:
I'd like to see an example added here, for perhaps a two phase system. The analytical solution to a two-phase problem would be nice here (and could be an additional regression test problem)